dandi/cli/cmd_download.py

```diff
@@ -11,16 +11,41 @@
 from ..download import DownloadExisting, DownloadFormat, PathType
 from ..utils import get_instance, joinurl
 
+_examples = """
+EXAMPLES: \n
+# Download only the dandiset.yaml \n
+dandi download --download dandiset.yaml DANDI:000027 \n
+
+# Download only dandiset.yaml if there is a newer version \n
+dandi download https://identifiers.org/DANDI:000027 --existing refresh
+
+# Download only the assets \n
+dandi download --download assets DANDI:000027
+
+# Download all from a specific version \n
+dandi download DANDI:000027/0.210831.2033
+
+# Download a specific file or directory \n
+dandi download dandi://DANDI/000027@0.210831.2033/sub-RAT123/sub-RAT123.nwb
```

Comment:
- what could trailing empty spaces would do besides in rare cases possibly negatively effecting wrapping?

- also why bother with extra new line?

- best to provide specific example for a directory since needs to end with `/` IIRC (please check that works)

```suggestion
EXAMPLES:\n
# Download only the dandiset.yaml
dandi download --download dandiset.yaml DANDI:000027

# Download only dandiset.yaml if there is a newer version
dandi download https://identifiers.org/DANDI:000027 --existing refresh

# Download only the assets
dandi download --download assets DANDI:000027

# Download everything for a specific version of a dandiset
dandi download DANDI:000027/0.210831.2033

# Download a specific directory
dandi download dandi://DANDI/000027@0.210831.2033/sub-RAT123/

# Download a file
dandi download dandi://DANDI/000027@0.210831.2033/sub-RAT123/sub-RAT123.nwb
```

Reply:
> what could trailing empty spaces would do besides in rare cases possibly negatively effecting wrapping?

I just used them to separate the \n for human reading, but I dont feel strongly. Removed in latest commit.

> also why bother with extra new line?

I'm not sure why, but a single newline is skipped, maybe something to do with the fstring? https://github.com/dandi/dandi-cli/blob/15196a93310618f8897c7b43444e216bbb094549/dandi/cli/cmd_download.py#L15-L16

Your suggestion renders like this:

```
  EXAMPLES:

  # Download only the dandiset.yaml dandi download --download dandiset.yaml
  DANDI:000027 # Download only dandiset.yaml if there is a newer version dandi
  download https://identifiers.org/DANDI:000027 --existing refresh # Download
  only the assets dandi download --download assets DANDI:000027 # Download
  everything for a specific version of a dandiset dandi download
  DANDI:000027/0.210831.2033 # Download a specific directory dandi download
  dandi://DANDI/000027@0.210831.2033/sub-RAT123/ # Download a file dandi
  download dandi://DANDI/000027@0.210831.2033/sub-RAT123/sub-RAT123.nwb
```